cumulusci/tasks/github/publish.py

```diff
@@ -221,15 +213,26 @@ def _create_release(self, path, commit):
             tagger={
                 "name": self.github_config.username,
                 "email": self.github_config.email,
-                "date": "{}Z".format(datetime.utcnow().isoformat()),
+                "date": f"{datetime.utcnow().isoformat()}Z",
```

Comment:
🤔: Well, no need to do this, but I wonder if we can just pass `datetime.now(timezone.utc).isoformat()` instead.